.github/actions/smoke-test/test.sh

```
@@ -1,16 +1,25 @@
 #/bin/bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)